client/verta/verta/_cli/multiple_arguments.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+
+class MultipleArguments:
```

Comment:
It looks like every time this is used, it is created and then immediately `for_each()` is called. Could this whole class be turned into one function, instead?

Reply:
Yes, you are right. Replaced to single function and moved to utils.